modules/coreos-layering-configuring.adoc

```diff
@@ -0,0 +1,191 @@
+// Module included in the following assemblies:
+//
+// * post-installation_configuration/coreos-layering.adoc
+
+:_content-type: PROCEDURE
+[id="coreos-layering-configuring_{context}"]
+= Applying a {op-system} custom layered image
+
+You can easily configure {op-system-first} image layering on the nodes in specific machine config pools. The Machine Config Operator (MCO) reboots those nodes with the new custom layered image, overriding the base {op-system-first} image.
+
+To apply a custom layered image to your cluster, you need to have the image in a repository that your cluster can reach. Then, create a `MachineConfig` object that points to the custom layered image. You need a separate `MachineConfig` object for each machine config pool that you want to configure.
+
+[IMPORTANT]
+====
+When you configure a custom layered image, {product-title} no longer automatically updates any node that uses the custom layered image. You become responsible to manually update your nodes as appropriate. If you roll back the custom layer, {product-title} will again automatically update the node. See the Additional resources section that follows for important information about updating nodes that use a custom layered image.
+====
+
+.Prerequisites
+
+* You must create a custom layered image that is based on an {product-title} image digest, not a tag.
++
+[NOTE]
+====
+You should use the same base {op-system} image that in installed on the rest of your cluster. Use the `oc adm release info --image-for rhel-coreos-8` command to obtain the base image being used in your cluster.
```

Comment:
Maybe it should be "image that **is** installed" instead of "image that **in** installed"